source/fundamentals/builders/aggregation-expression-operations.txt

```diff
@@ -694,73 +813,80 @@ the Query API:
+     - :manual:`$millisecond </reference/operator/aggregation/millisecond/>`
+
+   * - `minute() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#minute(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$minute </reference/operator/aggregation/minute/>`
+
+   * - `month() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#month(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$month </reference/operator/aggregation/month/>`
+
+   * - `second() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#second(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$second </reference/operator/aggregation/second/>`
+
+   * - `week() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#week(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$week </reference/operator/aggregation/week/>`
+
+   * - `year() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#year(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$year </reference/operator/aggregation/year/>`
 
 .. include:: /includes/fundamentals/agg-mql-diff.rst
 
-Suppose you have a collection of customers with their membership information.
-Originally, customers were either members or not. Over time, membership levels
-were introduced and used the same field. The information stored in this field
-can be one of a few different types, and you want to create a consistent field
-indicating their membership level.
+Suppose you have data about package deliveries and need to match
+deliveries that occurred on any Monday in the ``"America/New_York"`` time
+zone.
+
+If the ``deliveryDate`` field contains any string values representing
+valid dates, such as ``"2018-01-15T16:00:00Z"`` or ``Jan 15, 2018, 12:00
+PM EST``, you can use the ``parseDate()`` method to convert the strings
+into date types.
 
-To change what you are doing based on the data type of ``member``, use the
-``switchOn()`` method. If the data type is a string, it indicates the
-membership level and you want to use the data without altering it. If the data
-type is ``Boolean``, check the value with the ``cond()`` method and return a
-string depending on the result. The default membership level is "Gold". If the
-data type is ``Array``, it lists all of the customer's subscriptions, with the
-last one in the array being the current membership. Get the last element using
-the ``last()`` method. If it is not one of these types, label them as a "Guest"
-by using ``defaults()``. Regardless of the type, use the ``project()`` method
-to output the result as ``membershipLevel``.
+The ``dayOfWeek()`` method determines which day of the week it is and checks
+if it's a Monday by comparing it to ``2`` using the ``eq()`` method. The
```

Comment:
Question:
Does dayOfWeek() actually check both those things? If not, use separate sentences to separate the subject.